src/Alert.js

```diff
@@ -39,6 +65,11 @@ class Alert extends React.Component {
         className={classNames(className, classes)}
       >
         {dismissable && <CloseButton onClick={onDismiss} label={closeLabel} />}
+        {!!headingText && (
+          <HeadingComponent className="alert-heading">
+            {headingText}
+          </HeadingComponent>
```

Comment:
lets split this out into another component `AlertHeading`, like we do for Modal, ModalHeader, etc.